gobblin-dataset-config/src/test/resources/filebasedConfig/v1.0.1/application.conf

```diff
@@ -0,0 +1,40 @@
+// circular dependency for config-tag.t1.t2.t3 -> config-tag.tag1.tag2 -> config-tag.t1.t2.t3
```

Comment:
I assume these files are included in this PR by mistake. I don't see any class using them.